modules/ztp-image-based-upgrade-creating-backup-resources.adoc

```diff
@@ -0,0 +1,252 @@
+  annotations:
+    lca.openshift.io/apply-wave: "1"
+spec:
+  backupName:
+    backup-acm-klusterlet
+----
+<1> The `acm-klusterlet` `Backup` and `Restore` CRs are specific to {rh-rhacm} environments only.
+
+[source,yaml]
+----
+apiVersion: velero.io/v1
+kind: Restore
+metadata:
+  name: restore-example-app
+  namespace: openshift-adp
+  labels:
+    velero.io/storage-location: default
+  annotations:
+    lca.openshift.io/apply-wave: "2"
+spec:
+  backupName:
+    backup-example-app
+----
+
+[NOTE]
+====
+Depending on your {rh-rhacm} configuration, you might need to include the `v1/secrets/open-cluster-management-agent/open-cluster-management-image-pull-credentials` object  for backup. If your `multiclusterHub` CR has `.spec.imagePullSecret` defined and the secret exists on the `open-cluster-management-agent` namespace in your hub cluster, ensure that the object is included in the `lca.openshift.io/apply-label` annotation. If the secret does not exist, you can remove the object from the apply-label annotation.
+====
+
+[IMPORTANT]
+====
+To use the `lca.openshift.io/apply-label` annotation for backing up specific resources, the resources listed in the annotation should also be included in the `spec` section.
+If the `lca.openshift.io/apply-label` annotation is used in the `Backup` CR, only the resources listed in the annotation will be backed up, even if other resource types are specified in the `spec` section or not.
+====
+--
+
+.. Define the apply order for the OADP Operator in the `Restore` CRs by using the `lca.openshift.io/apply-wave` field:
++
+--
+.Example OADP CRs
+[source,yaml]
+----
+apiVersion: velero.io/v1
+kind: Backup
+metadata:
+  labels:
+    velero.io/storage-location: default
+  name: backup-example-app
+  namespace: openshift-adp
+spec:
+  includedNamespaces:
+  - test
+  includedNamespaceScopedResources:
+  - secrets
+  - persistentvolumeclaims
+  - deployments
+  - statefulsets
+  excludedClusterScopedResources:
+  - persistentVolumes
+----
+
+[source,yaml]
+----
+apiVersion: velero.io/v1
+kind: Restore
+metadata:
+  name: restore-acm-klusterlet
+  namespace: openshift-adp
+  labels:
+    velero.io/storage-location: default
+  annotations:
+    lca.openshift.io/apply-wave: "1"
+spec:
+  backupName:
+    backup-acm-klusterlet
+----
+
+[source,yaml]
+----
+apiVersion: velero.io/v1
+kind: Restore
+metadata:
+  name: restore-example-app
+  namespace: openshift-adp
+  labels:
+    velero.io/storage-location: default
+  annotations:
+    lca.openshift.io/apply-wave: "2"
+spec:
+  backupName:
+    backup-example-app
+----
+
+[NOTE]
+====
+If you do not define the `lca.openshift.io/apply-wave` annotation in the `Backup` or `Restore` CRs, they will be applied together.
+====
+--
+
+. Generate a `ConfigMap` object for your OADP CRs.
+
+.. Create the `ConfigMap` object:
++
+[source,terminal]
+----
+$ oc create configmap example-oadp-cm --from-file=example-oadp-resources.yaml=<path_to_oadp_crs> -n openshift-adp
```

Comment:
I am thinking maybe it would be clearer if you could have two example yamls 
- (optional for acm managed cluster) acmKlusterletBackupRestore.yaml contains backup and restore CRs with separator "---" in between for acm resources
- exampleAppBackupRestore.yaml contains backup and restore CRs with separator ---" in between for app resources

The command would be something like 
`oc create configmap example-oadp-cm  --from file=exampleAppBackupRestore.yaml=exampleAppBackupRestore.yaml (--from-file=acmKlusterletBackupRestore.yaml=acmKlusterletBackupRestore.yaml) -n openshift-adp`

A common description for `lca.openshift.io/apply-wave` can be added to describe that's for defining the apply order for dependent resources.